zcash_client_backend/proto/service.proto

```diff
@@ -8,18 +8,31 @@ option go_package = "lightwalletd/walletrpc";
 option swift_prefix = "";
 import "compact_formats.proto";
 
+// An identifier for a Zcash value pool.
+enum PoolType {
+    TRANSPARENT = 0;
```

Comment:
According to best practices: https://protobuf.dev/best-practices/dos-donts/#unspecified-enum enums should include an unspecified value. This probably is because zero is the default value you never get a default value by accident.

Reply:
Thanks, I hate it. (Yes we should do this. See https://protobuf.dev/programming-guides/enum/ for why we don't want a closed enum, therefore we have to define the useless "unspecified" value.)

Reply:
Fixed in https://github.com/zcash/lightwallet-protocol/pull/1